docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -34,6 +34,27 @@ appropriate for your setup:
     connection-user=exampleuser
     connection-password=examplepassword
 
+.. _clickhouse-tls:
+
+Connection security
+^^^^^^^^^^^^^^^^^^^
+
+If you have TLS configured with a globally-trusted certificate installed on your
+data source, you can enable TLS between your cluster and the data
+source with a JDBC URL parameter in the ``connection-url`` catalog configuration
+property.
+
+For example, with version 2.6.4 of the ClickHouse JDBC driver, enable TLS by
+appending the ``ssl=true`` parameter to the ``connection-url`` configuration
+property:
```

Comment:
OK, super-nitty, here and throughout: 
You're really appending a parameter to the JDBC connection string. It is that string which is entered as the argument to the connection-url property. 
No pixels will die if you leave it as-is. Just being proper here.

Reply:
No that's fair, I would rather be precise about referring to the JDBC connection string. Edited.